layers/+lang/ocaml/packages.el

```diff
@@ -41,6 +40,8 @@
       :defer t
       :init
       (progn
+        (add-hook 'tuareg-mode-hook 'flycheck-mode)
```

Comment:
I don't think this is necessary as we already have `(spacemacs/enable-flycheck 'tuareg-mode)`

Reply:
Ah, it didnt seem to kick in but I'd missed that - will take another look and see if I can clean it up.